apps/workspaces/tasks.py

```
@@ -134,18 +154,19 @@ def async_add_admins_to_workspace(workspace_id: int, current_user_id: str):
         fyle_user = get_fyle_admin(access_token.split(' ')[1], None)
         org_name = fyle_user['data']['org']['name']
 
         workspace.name = org_name
         workspace.save()
     except InvalidTokenError:
-        logger.info("Invalid Token for Fyle in workspace_id: %s", workspace.id)
+        logger.info("Invalid Token for Fyle in workspace_id: %s", workspace_id)
 
     except Exception as e:
-        logger.exception("Error updating workspace name for workspace_id: %s | Error: %s", workspace.id, str(e))
+        logger.exception("Error updating workspace name for workspace_id: %s | Error: %s", workspace_id, str(e))
```

Ruff (0.14.13)

Redundant exception object included in `logging.exception` call

(TRY401)